endorse/commit.go

```diff
@@ -284,25 +291,22 @@ func snapshotEndorsement(ctx context.Context, cops ChangeOps, endorsement *epb.V
 	if err != nil {
 		return err
 	}
+	binPaths := []string{fwPath, fwEvtsPath}
 	files := []*File{
 		{Path: fwPath, Contents: ec.Image},
-		{Path: evtsPath, Contents: events},
+		{Path: fwEvtsPath, Contents: events},
 	}
 	if len(ec.SvsmImage) != 0 {
-		files = append(files, &File{Path: svsmPath, Contents: ec.SvsmImage})
+		files = append(files, &File{Path: svsmPath, Contents: ec.SvsmImage},
+			&File{Path: svsmEvtsPath, Contents: events})
+		binPaths = append(binPaths, svsmPath, svsmEvtsPath)
 	}
 	if err := cops.WriteOrCreateFiles(ctx, files...); err != nil {
 		return err
 	}
-	if err := cops.SetBinaryWritable(ctx, fwPath); err != nil {
-		return fmt.Errorf("could not set %q type to binary: %w", fwPath, err)
-	}
-	if err := cops.SetBinaryWritable(ctx, evtsPath); err != nil {
-		return fmt.Errorf("could not set %q type to binary: %w", evtsPath, err)
-	}
-	if len(ec.SvsmImage) != 0 {
-		if err := cops.SetBinaryWritable(ctx, svsmPath); err != nil {
-			return fmt.Errorf("could not set %q type to binary: %w", svsmPath, err)
+	for _, path := range binPaths {
```

Comment:
Can't you reuse the files slice instead of making binPaths as well?

Reply:
Done.